charts/feathr-online/templates/configmap.yaml

```diff
@@ -4,7 +4,16 @@ metadata:
   name: feathr-online-config
   namespace: {{ .Release.Name }}
 data:
-  pipeline.conf: {{ .Files.Get ( required ".Values.pipelineConf is required." .Values.pipelineConf) | indent 4 }}
+  # pipeline.conf and lookup.json needs to be encoded with Base64
```

Comment:
Please add a comment where so people know decode is needed to workaround any path issue. Better add the reference link.